src/Tests/SenseNet.Packaging.Tests/SafeQueries.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using SenseNet.Search;
+
+namespace SenseNet.Packaging.Tests
+{
+    internal class SafeQueries : ISafeQueryHolder
+    {
```

Comment:
SafeQuery feature can be replaced with the CreateSafeQuery method of the TestBase.